### ISSUE TITLE
When preset is to the current preset, if still asks me to delete

### ISSUE THREAD

**Issue by gbraad** [crc-org/crc-extension#130]
1. start with microshift preset
2. change to openshift preset
3. answer 'cancel' to the preset changed dialog
4. change back to microshift
5. still get the change dialog while this is the same as the current (running) preset

**Comment by gbraad** [comment]
I guess 

https://github.com/crc-org/crc-extension/blob/f975968de77c148687b4165332c7e2d244b0ff73/src/preferences.ts#L147-L148

`let needRecreateCrc = false;`

is not properly set.

**Comment by gbraad** [comment]
Added:
```
  // check for recreate need based on current status too
  if (currentStatus.Preset !== newConfig.preset) {
    console.log("Preset doesn't match. Need to recreate");
    needRecreateCrc = true;
    needRefreshConfig = true;
  }
  else {
    console.log("Preset matches. No need to recreate");
    needRecreateCrc = false;
    needRefreshConfig = true;
  }
```

but experienced issues with a crashing VM/sshd inside the VM.

**Comment by gbraad** [comment]
Fixed in v1.0.3
